src/AudioIO.cpp

```diff
@@ -1463,31 +1463,39 @@ void AudioIO::StopStream()
       // call TrackBufferExchange one last time (it normally would not do so since
       // Pa_GetStreamActive() would now return false
       mAudioThreadShouldCallTrackBufferExchangeOnce
-         .store(true, std::memory_order_relaxed);
-      while( mAudioThreadShouldCallTrackBufferExchangeOnce
-         .load(std::memory_order_relaxed) )
+         .store(true, std::memory_order_release);
+      while (mAudioThreadShouldCallTrackBufferExchangeOnce
+         .load(std::memory_order_acquire))
```

Comment:
This is still incomplete without changing the uses of this atomic in `AudioThread::Entry` too, with more than
relaxed ordering.

Lock-free synchronization with an atomic variable requires both communicating threads to use the right memory orderings.